requirements.txt

```diff
@@ -1,4 +1,5 @@
+bump2version==1.0.0
 datadog==0.36.0
 pytest==5.4.2
-bump2version==1.0.0
+requests>=2.20.1
```

Comment:
The version pin is stricter on setup.py than on requirements.txt . This can cause version conflicts in the future when the requests lib updates.